tests/data/power_flow/automatic-tap-regulator/auto-tap-changer-meshed-any-max-iter/params.json

```diff
@@ -3,7 +3,7 @@
   "tap_changing_strategy": "any_valid_tap",
   "rtol": 1e-05,
   "atol": 1e-05,
-  "fail": {
+  "raises": {
```

Comment:
this one may be up for discussion on whether it should be `raises` or `xfail`...

Reply:
alright